lib/hca/military_information.rb

```diff
@@ -5,12 +5,21 @@
 module HCA
   class MilitaryInformation
     PREFILL_METHODS = %w[
+      currently_active_duty
+      currently_active_duty_hash
+      discharge_type
+      guard_reserve_service_history
```

Comment:
why was all this stuff added to the `HCA::MilitaryInformation` class? this class is supposed to only be used only by the health care application

Reply:
At first we separated it out into it's own class, but we thought it made more sense to put all the prefill methods in one class.

Reply:
That's fine with me but would it be possible to move the whole file to a different namespace? As far as I know, all the code in the `HCA` namespace is only used by the health care application.

Reply:
@lihanli Good idea! We implemented the changes you suggested. Can you take another look?